servatrice/src/servatrice.cpp

```diff
@@ -566,6 +565,11 @@ void Servatrice::statusUpdate()
         return;
 
     const int uc = getUsersCount(); // for correct mutex locking order
+
+    const QList<QString> mods_info = getOnlineModeratorList();
```

Comment:
use `QStringList` instead of `QList<QString>`, add the #include if needed

Reply:
Seems silly as they should be equiv... but w/e i'll get that fixed